### PR TITLE
Tests for Game and ChooseWonder services implemented

### PR DESCRIPTION
**Pull request by gicck**

### CODE REVIEW

File: ui/src/main/javascript/app/SpecRunner.html [hunk 1]
```diff
@@ -45,14 +45,15 @@
 	<script src="/lobby/lobby.module.js"></script>
 	<script src="/lobby/lobby.component.js"></script>
 	<script src="/gameroom/gameroom.component.js"></script>
-	<script src="/gameboard/gameboard.module.js"></script>
-	<script src="/gameboard/gameboard.component.js"></script>
+	<!--<script src="/gameboard/gameboard.module.js"></script>-->
```
Comment (ziur) [inline ui/src/main/javascript/app/SpecRunner.html]: Why are you commenting these lines?

**Comment by jjohxx** [comment]
It looks good to me if you remove unnecessary dependencies.

**Review by jjohxx — APPROVED** [review]

File: ui/src/main/javascript/app/core/choosewonder/choosewonder.service.js [hunk 2]
```diff
@@ -4,29 +4,29 @@ angular.
     module('sevenWonders.core.choosewonder').
     factory('ChooseWonder', ['$cookies', '$websocket', 'Game', 'Restangular', '$q', '$location',
```
Comment (jjohxx) [inline ui/src/main/javascript/app/core/choosewonder/choosewonder.service.js:5]: You must remove unnecessary dependencies like to be $websocket, $location at the same time the parameters of the function.

File: ui/src/main/javascript/app/core/choosewonder/choosewonder.service.js [hunk 3]
```diff
@@ -4,29 +4,29 @@ angular.
...
+                                resolve(data);
+                            }).catch(function (data) {
+                                reject(data);
+                            });
+                    });
+                },
 
-                    getWonderPlayers : function () {
-                       return $q(function(resolve, reject){
+                getWonderPlayers: function () {
```
Comment (jjohxx) [inline ui/src/main/javascript/app/core/choosewonder/choosewonder.service.js]: I understand that this name is not general, we are here getting all the players of a specific game. This should have another name and not getWonderPlayers. I think it ...

**Review by jjohxx — APPROVED** [review]

**Review by ziur — APPROVED** [review]

**Pull request merged**